src/host/ut_host/ConsoleArgumentsTests.cpp

```diff
@@ -90,7 +90,7 @@ void ConsoleArgumentsTests::ArgSplittingTests()
                    INVALID_HANDLE_VALUE,
                    INVALID_HANDLE_VALUE,
                    ConsoleArguments(commandline,
-                                    L"this is the commandline", // clientCommandLine
+                                    L"\"this is the commandline\"", // clientCommandLine

```

Comment:
We should still maintain a test variant that does it without the quotes in it so we can ensure that the non-quoted string is doing the appropriate thing too.

Reply:
I added a test without quotes.